scripts/main/settings.js

```diff
@@ -4,130 +4,129 @@
 
 let settings = {};
 
+/**
+ * @returns {void}
+ */
 settings.open = function () {
 	view.settings.init();
 };
 
+/**
+ * @returns {void}
+ */
 settings.createConfig = function () {
+	/**
+	 * @param {XMLHttpRequest} jqXHR the jQuery XMLHttpRequest object, see {@link https://api.jquery.com/jQuery.ajax/#jqXHR}.
+	 * @param {Object} params the original JSON parameters of the request
+	 * @param {?LycheeException} lycheeException the Lychee exception
+	 * @returns {boolean}
+	 */
+	const errorHandler = function (jqXHR, params, lycheeException) {
+		const _data = lycheeException ? lycheeException.message : lychee.locale["ERROR_UNKNOWN"];
+
+		// TODO: Where, when and how does the server throw these error messages?
+		// Have these error message ever been used? The backend doesn't
+		// define these message and had not even done so before the re-factoring.
+		// Are these conditions legacy and probably never taken?
```

Comment:
Legacy Lychee v3: 
The initial version of Lychee was connecting to the MySql server and creating the DB.
This is no longer the case because we now have /install
So this should be removed.

Reply:
I nuked it completely.